src/databricks/labs/ucx/cli.py

```diff
@@ -705,8 +705,14 @@ def migrate_acls(
         workspace_contexts = [ctx]
     else:
         workspace_contexts = _get_workspace_contexts(w, a, run_as_collection, **named_parameters)
-    target_catalog, hms_fed = named_parameters.get("target_catalog"), named_parameters.get("hms_fed", False)
+    target_catalog, hms_fed, dry_run = (
+        named_parameters.get("target_catalog"),
+        named_parameters.get("hms_fed", False),
+        named_parameters.get("dry_run", False),
+    )
     for workspace_context in workspace_contexts:
+        if dry_run:
+            workspace_context.acl_migrator.snapshot()
```

Comment:
and what do we do with it?

Reply:
That will be displayed in a dashboard.

Reply:
Coming in a follow up PR

Reply:
in the current shape, it just snapshots the data and continues to migrate ACLs - **not doing what this PR intends to do**, introducing a bug.

also user is not presented with any results of a dry run. no log, no whatever.

<img width="1234" alt="image" src="https://github.com/user-attachments/assets/b8b511da-e386-4090-8e22-8f6738c57edf">